docs: Add new blog post APISIX deployment on Scaleway cloud (#1365)
iresty/iresty.github.io: blog/en/blog/2022/10/19/deploy-apisix-on-scaleway-cloud.md

Firstly, let’s understand what is Apache APISIX and Scaleway, how you can leverage both solutions to serve your needs. Or you can just skip the intro sections and start with the tutorial.

## What’s Apache APISIX?

[Apache APISIX](https://apisix.apache.org/) is an open-source  **API Gateway** under [Apache Software Foundation](https://www.apache.org/) that is lightweight, independently deployable, and scalable that can run anywhere that allows developers to manage API endpoints. You can also leverage APISIX as [Kubernetes Ingress Controller](url) to deliver high performance and get the benefits of stateful load balancing, traffic split, hot reloading, and expansion capabilities by means of its offered diverse plug-ins to satisfy your specific needs.  You can read more about the use cases and features offered by Apache APISIX in the [documentation](https://apisix.apache.org/docs/apisix/getting-started/).

> ![photo_2022-04-03_22-38-23.jpg](https://static.apiseven.com/2022/10/17/634d545743b1a.jpg)

Reviewer: ```suggestion
![Apache APISIX](https://static.apiseven.com/2022/10/17/634d545743b1a.jpg)
```

> It is also possible to install Apache APISIX by [different method](https://apisix.apache.org/docs/apisix/installation-guide/)s ([Docker](https://www.docker.com/), [Helm](https://helm.sh/), or [RPM](https://rpm.org/)) and run it in the various public cloud providers because of its [cloud-native](https://learn.microsoft.com/en-us/dotnet/architecture/cloud-native/definition) behavior.

Reviewer: ```suggestion
It is also possible to install Apache APISIX by [different methods](https://apisix.apache.org/docs/apisix/installation-guide/) ([Docker](https://www.docker.com/), [Helm](https://helm.sh/), or [RPM](https://rpm.org/)) and run it in the various public cloud providers because of its [cloud-native](https://learn.microsoft.com/en-us/dotnet/architecture/cloud-native/definition) behavior.
```

Reviewer: <img width="1118" alt="image" src="https://user-images.githubusercontent.com/39793568/196615620-08379740-e282-4630-8fb1-353606845ed8.png">

Author: @SylviaBABY Very good catch! Thanks.

apisix-5bcf68b548-qrsqb                      1/1     Running   0          1m
apisix-etcd-0                                1/1     Running   0          1m
apisix-etcd-1                                1/1     Running   0          1m
apisix-etcd-2                                1/1     Running   0          1m
apisix-ingress-controller-75bd4d9b9b-7xfn5   1/1     Running   0          1m

$ kubectl get svc -n ingress-apisix

apisix-admin                ClusterIP      10.36.120.143   <none>          9180/TCP            82m
apisix-etcd                 ClusterIP      10.47.39.201    <none>          2379/TCP,2380/TCP   82m
apisix-etcd-headless        ClusterIP      None            <none>          2379/TCP,2380/TCP   82m
apisix-gateway              LoadBalancer   10.45.241.11    51.159.206.46   80:30197/TCP        82m
apisix-ingress-controller   ClusterIP      10.47.236.40    <none>          80/TCP              82m
```

Afterward, we can deploy mock server `htttbin` to the default namespace in order to test the ingress controller:

``` shell
kubectl run httpbin --image kennethreitz/httpbin --port 80
kubectl expose pod httpbin --port 80
```

> Now we are ready to test ingress by creating a new route the same as below

Reviewer 2: ```suggestion
Now we are ready to test ingress by creating a new route the same as below:
```

``` shell
kubectl apply -f - <<EOF

> apiVersion: apisix.apache.org/v2beta3

Reviewer 3: we have released APISIX Ingress v1.5, we should change this apiVersion to v2
```suggestion
apiVersion: apisix.apache.org/v2
```

Author: @tao12345666333 Right!
